tests/module_api/test_account_data_manager.py

```diff
@@ -110,3 +110,49 @@ def test_get_global_no_mutability(self) -> None:
             ),
             {"wombat": True},
         )
+
+    def test_put_global(self) -> None:
+        """
+        Tests that written account data using `put_global` can be read out again later.
+        """
+
+        self.get_success(
+            self._module_api.account_data_manager.put_global(
+                self.user_id, "test.data", {"wombat": True}
+            )
+        )
+
+        # Request that account data from the normal store; check it's as we expect.
+        self.assertEqual(
+            self.get_success(
+                self._store.get_global_account_data_by_type_for_user(
+                    self.user_id, "test.data"
+                )
+            ),
+            {"wombat": True},
+        )
+
+    def test_put_global_validation(self) -> None:
```

Comment:
Should we also test bad data types for these parameters?

Reply:
Good idea